modules/sno-du-configuring-the-container-mountspace.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-the-container-mountspace_{context}"]
+= Configuring the container mount namespace
+
+To reduce the overall management footprint of the platform, a machine configuration is provided to contain the mount points.
+
+.Procedure
```

Comment:
When you say the machine config _is provided_, does the user need to do anything?
Or is is that the machine config is provided in this document. You need to create it in your cluster. 
 
If not, we don't need the .Procedure.

Reply:
Removed .Procedure